src/icalendar/prop.py

```diff
@@ -719,7 +719,7 @@ def __new__(cls, value, encoding=DEFAULT_ENCODING):
         return self
 
     def __repr__(self):
-        return f"vText('{self.to_ical()}')"
+        return f"vText('{self.to_ical()!r}')"
```

Comment:
Depending on desired outcome, another fix would be
```suggestion
        return f"vText('{self.to_ical().decode(self.encoding)}')"
```

Reply:
I like the `decode` version better